src/main/java/com/capstone/producer/TfrHandler.java

```diff
@@ -0,0 +1,181 @@
+        Matcher matcher = pattern.matcher(notam);
+        
+
+        // if matcher has a string, we have a multipart notam.
+        // if not, it is a single part notam
+        if(matcher.find()){
+            LOGGER.debug("Processing TFR NOTAM PART {} of {}", matcher.group(2), matcher.group(4));
+            if(arryBuilder == null){
+                arryBuilder = new String[Integer.parseInt(matcher.group(4))];
+            }
+            arryBuilder[Integer.parseInt(matcher.group(2)) - 1] = notam;
+            receivedNotams.put(notamNumber, arryBuilder);
+            // test if array is filled out
+            for(int i = 0; i < arryBuilder.length; i++) {
+                if(arryBuilder[i] == null)
+                    return false;
+            }
+            return true;
+        }
+        // single message NOTAM
+        arryBuilder = new String[1];
+        arryBuilder[0] = notam;
+        receivedNotams.put(notamNumber, arryBuilder );
+        return true;
+    }
+
+    private static String combineNotamMessages(String notamNumber) {
+        String[] allMessages = receivedNotams.get(notamNumber);
+        String fullMessage = "";
+        Pattern pattern = Pattern.compile("(FDC\\s*\\d/\\d{4}\\s*.*..TEMPORARY\\s*(FLIGHT)?\\s*(RESTRICTIONS.)?)(.*)(\\b\\d{10}-\\d{10})");
+        String stringToParse = "";
+        for(int i = 0; i < allMessages.length; i++) {
+            LOGGER.debug("AllMessage of message: {}", allMessages[i]);
+            stringToParse = allMessages[i].replaceAll(System.getProperty("line.separator"), " ");
+            stringToParse = stringToParse.replaceAll("[\\n\\r]", " ");
+            LOGGER.debug("String to parse: {}", stringToParse);
+            Matcher matcher = pattern.matcher(stringToParse);
+            if(matcher.find()) {
+                fullMessage += matcher.group(4);
+            }
+        }
+        return fullMessage;
+    }
+
+    private static boolean pointRadiusParse(String notamNumber, String message) throws InterruptedException, JsonProcessingException {
+        Pattern pattern = Pattern.compile("(WI\\s*AN\\s*AREA\\s*DEFINED\\s*AS\\s*(\\d*.?\\d*)NM\\s*RADIUS\\s*OF\\s*(\\d+[NS]\\d+[EW])(.*?)EFFECTIVE\\s*(\\d{10})\\s*UTC.*?UNTIL\\s*(\\d{10})?)");
+        Matcher matcher = pattern.matcher(message);
+        ObjectMapper objectMapper = new ObjectMapper();
+        boolean successfulMatching = false;
+        while(matcher.find()){ //Allows for multiple finds.
+            LOGGER.debug("String matched Radius test: {}", matcher.group(0));
+
+            String endString;
+            if(matcher.group(6) != null) {
+                endString = matcher.group(6);
+            } else {
+                endString = "PERM";
+            }
+
+            List<String> latlong = new ArrayList<String>();
+            latlong.add(matcher.group(3));
+            
+            TfrNotam notamObject = new TfrNotam(notamNumber, "RADIUS", latlong, matcher.group(2), matcher.group(5), endString);
```

Comment:
Fluent setters have been configured for this project. We should prefer using them because they make the code easier to read by explicitly stating what is being assigned to what. Also, I think sonar flags constructors with more than 7 arguments. 

Don't bother fixing it in this pull request thought.